Server/src/services/getBasicDataService.ts

```diff
@@ -0,0 +1,28 @@
+import { Connection, getConnection } from "typeorm";
+import { selectAllCategoriesQuery } from "../models/entities/Category";

```

Comment:
Why are we importing here from the entity inside a service?

Reply:
I was under a mistaken impression that it wasn't bad practice but has now been corrected.

Reply:
Its not that its bad practice is that we are trying to keep the modularity